gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ConditionMapper.java

```diff
@@ -152,6 +148,14 @@ private Condition getCondition(Patient patient, List<Encounter> encounters, RCMR
         return condition;
     }
 
+    private Condition initializeCondition(String id, String practiceCode, Meta meta) {
+        return (Condition) new Condition()
+            .addIdentifier(buildIdentifier(id, practiceCode))
+            .addCategory(generateCategory())
+            .setId(id)
+            .setMeta(meta);
+    }
```

Comment:
Why can't this just be a constructor within `Condition` itself, thinking of **S**OLID! 😄